text/0000-repr-transparent.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: `repr_transparent`
+- Start Date: 2016-09-26
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Extend the existing `#[repr]` attribute on newtypes with a `transparent` option
+specifying that the type representation is the representation of its only field.
```

Comment:
`#[repr(C)]` can also influence how values are passed to and returned from FFI functions, notably `struct Foo(f64)` on ARM64.